juniper_official/interface/collect-physical-interface-stats.rule

```diff
@@ -23,6 +24,13 @@
                     frequency 60s;
                 }
             }
+            sensor cisco-oc-ifd {
+                description "Cisco open-config Physical interface sensor definition";
+                open-config {
+                    sensor-name /interfaces/interface/state/counters;
```

Comment:
I think it's better to subscribe to /interfaces/interface/state/. Please sync up with Vikram as there will be observability rules on this, we would like to align from product perspective to minimise data collection.